rtos/rtx2/TARGET_CORTEX_M/TARGET_M0P/TOOLCHAIN_GCC/irq_cm0.S

```diff
@@ -24,7 +24,7 @@
  */
 
 
-        .file    "irq_cm0.S"
+        .file    "irq_cm0.s"
```

Comment:
I don't know, I think it was the other way around we updated it from lower to upper, but it happened this way during history squash. Now it's fixed for cmsis but i remember it being broken https://github.com/ARM-software/CMSIS_5/commit/200cd4bbd7445c5471e30659b75380b17e4e7a8d

reverted.